targets/lenovo-x1-carbon.nix

```diff
@@ -30,6 +30,32 @@
         productId = "a7a1";
       }
     ];
+    audio.pciDevices = [
+      # Passthrough of Intel Audio device 8086:51ca
+      # With the current implementation, the whole PCI IOMMU group 14:
+      #   00:1f.x in the example from Lenovo X1 Carbon
+      #   must be defined for passthrough to AudioVM
+      {
+        path = "0000:00:1f.0";
+        vendorId = "8086";
+        productId = "5194";
+      }
+      {
+        path = "0000:00:1f.3";
+        vendorId = "8086";
+        productId = "51ca";
+      }
+      {
+        path = "0000:00:1f.4";
+        vendorId = "8086";
+        productId = "51a3";
+      }
+      {
+        path = "0000:00:1f.5";
+        vendorId = "8086";
+        productId = "51a4";
```

Comment:
Could you actually add comment here about what these devices are?

I went to X1 and with lspci I got:
```
00:1f.0 ISA bridge: Intel Corporation Raptor Lake LPC/eSPI Controller (rev 01)
00:1f.3 Audio device: Intel Corporation Raptor Lake-P/U/H cAVS (rev 01)
00:1f.4 SMBus: Intel Corporation Alder Lake PCH-P SMBus Host Controller (rev 01)
00:1f.5 Serial bus controller: Intel Corporation Alder Lake-P PCH SPI Controller (rev 01)
```

Reply:
Also another thing I'm a bit curious about. Did you try this with HDMI audio output? At least with the previous generations of Intel integrated GPUs, the sound card had to be passed through to the same VM as the graphics card, if you want the audio to work from HDMI output.

Reply:
> Could you actually add comment here about what these devices are?
> 
> I went to X1 and with lspci I got:
> 
> ```
> 00:1f.0 ISA bridge: Intel Corporation Raptor Lake LPC/eSPI Controller (rev 01)
> 00:1f.3 Audio device: Intel Corporation Raptor Lake-P/U/H cAVS (rev 01)
> 00:1f.4 SMBus: Intel Corporation Alder Lake PCH-P SMBus Host Controller (rev 01)
> 00:1f.5 Serial bus controller: Intel Corporation Alder Lake-P PCH SPI Controller (rev 01)
> ```

@mikatammi - it's literally commented here https://github.com/tiiuae/ghaf/pull/395/files#diff-789892fdf4e4c650d81ce42dbc84586807ced5e73a8d53c06775acf7df2d0876R34-R37 as I asked in already resolved comment.

Reply:
> Also another thing I'm a bit curious about. Did you try this with HDMI audio output? At least with the previous generations of Intel integrated GPUs, the sound card had to be passed through to the same VM as the graphics card, if you want the audio to work from HDMI output.

My 2 cents on scoping HDMI ARC testing out of the scope for this PR. It was never target for this iteration.

Reply:
I have not tried with HDMI output and you are probably correct about it. For now I think we don't care about hdmi output as putting GPU and audio directly to the same VM is not considered as an option.